src/ui/logo.rs

```diff
@@ -0,0 +1,137 @@
+use floem::cosmic_text::{Attrs, AttrsList, FamilyOwned, Style, TextLayout};
+use floem::peniko::Color;
+use floem::views::{rich_text, RichText};
+use std::ops::Range;
+
+const BANNER: &str = "
+ __   __    _     _   _   _      _____
+ \\ \\ / /   /_\\   | | | | | |    |_   _|
+  \\ V /   / _ \\  | |_| | | |__    | |
+   \\_/   /_/ \\_\\  \\___/  |____|   |_|
+   ";
+
+pub fn red_small_logo() -> RichText {
+	rich_text(move || {
+		let attrs_list = AttrsList::new(
+			Attrs::new()
+				.color(Color::RED)
+				.style(Style::Normal)
+				.family(&[FamilyOwned::Monospace])
+				.font_size(8.0),
+		);
+
+		let mut tl = TextLayout::new();
+		tl.set_text(BANNER, attrs_list);
+		tl
+	})
+}
+
+#[allow(dead_code)]
+pub fn rainbow_large_logo() -> RichText {
```

Comment:
Would be better to do:
```suggestion
pub fn _rainbow_large_logo() -> RichText {
```

That way we catch if there is dead code inside the function somewhere?